docs/concepts/improve_group_by_operation.md

```diff
@@ -21,13 +22,12 @@ The pandas API most likely cannot efficiently handle the complexity of the aggre
     df_pd = pd.DataFrame(data)
 
 
-    @nw.narwhalify
-    def approach_1(df: IntoFrameT) -> IntoFrameT:
-
+    def approach_1(df_native: IntoFrameT) -> IntoFrameT:
+        df = nw.from_native(df_native)
         # Pay attention to this next line
         df = df.group_by("a").agg(d=(nw.col("b") + nw.col("c")).sum())
```

Comment:
I know this was not introduced here, but instead of a comment we could use [mkdocsmaterial code highlighting](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#highlighting-specific-lines)